core/src/main/java/org/infinispan/interceptors/distribution/BaseDistributionInterceptor.java

```diff
@@ -358,44 +360,55 @@ public Object visitGetAllCommand(InvocationContext ctx, GetAllCommand command) t
          }
          return invokeNext(ctx, command);
       }
+      GetAllSuccessHandler getAllSuccessHandler = new GetAllSuccessHandler(command);
```

Comment:
Calling `visitGetAllCommand` would also mean that it would call the interceptors below, executing the command and then checking if it loaded all entries or throwing OTE (as write commands should). I think that the error handling was the main reason why I went for the refactoring.